src/client/src/Views/TemplateListView.tsx

```diff
@@ -72,7 +72,7 @@ export const TemplateListView = () => {
                   <td>{t.name}</td>
                   <td>{getOpeningDayDetails(t)}</td>
                   <td>
-                    <Link to={`edit/${t.id}`}>
+                    <Link href={`templates/edit/${t.id}`}>
```

Comment:
Is this change of path intentional. Assume it is because previous routes worked with relative as opposed to absolute

Reply:
It is and your assumption is correct, although I anticipated nextjs to be able to handle relative paths, it doesnt work without "templates/". Adding a leading slash doesnt seem to work, this may be because both /edit and edit/[:id] route to the same component though other sources indicate relative paths arent  supported: https://stackoverflow.com/questions/69084689/relative-routing-in-next-js